src/Domain/LeanCode.CQRS.Execution/LeanCode.CQRS.Execution.csproj

```
@@ -7,6 +7,7 @@
   <ItemGroup>
     <PackageReference Include="LeanCode.Contracts" />
     <PackageReference Include="Serilog" />
+    <PackageReference Include="Microsoft.Extensions.Hosting.Abstractions" Version="7.0.0" />
```

The version is already pinned [here](https://github.com/leancodepl/corelibrary/blob/v7.0/Directory.Build.targets#L84-L85).
```suggestion
    <PackageReference Include="Microsoft.Extensions.Hosting.Abstractions" />
```